connectors/sources/jira.py

```diff
@@ -784,11 +806,13 @@ async def _get_projects(self):
                 when=datetime.now(pytz.timezone(timezone))  # pyright: ignore
             )
             if self.jira_client.projects == ["*"]:
+                self._logger.info("Fetching all Jira projects")
                 async for response in self.jira_client.api_call(url_name=PROJECT):
                     response = await response.json()
                     for project in response:
                         await self._put_projects(project=project, timestamp=timestamp)
             else:
+                self._logger.info("Fetching the configured Jira projects")
```

Comment:
I think we could also include the projects here (maybe truncate them, if they're too many)